doc/src/manual/documentation.md

```diff
@@ -343,17 +343,17 @@ for your custom type that returns the documentation on a per-instance basis. For
 
 ```julia
 struct MyType
-    value::String
+    value
```

Comment:
I think this should still be concretely type, because having untyped fields is rather unidiomatic.
```suggestion
    value::Int
```